src/eureka/S5_lightcurve_fitting/s5_fit.py

```diff
@@ -647,6 +647,25 @@ def fit_channel(meta, time, flux, chan, flux_err, eventlabel, params,
                                          nints=lc_model.nints,
                                          num_planets=meta.num_planets)
         modellist.append(t_eclipse)
+    if 'fleck_tr' in meta.run_myfuncs:
```

Comment:
Looks like with the new implementation of AstroModel there's nothing I need to do to integrate the option to use fleck_tr with the PC models, I just need to add the model name to the fleck transit model so that the AstroModel code knows it's a transit